app/main.py

```diff
@@ -1,2 +1,91 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self, capacity: int = 8, load_factor: float = 0.7) -> None:
+        self.capacity = capacity
+        self.load_factor = load_factor
+        self.size = 0
+        self.table = [None] * capacity
+
+    def _hash(self, key: Any) -> int:
```

Comment:
key can't be Any, it must be Hashable type ( you can import from typing lib)
In all places where you waiting for Any you can change it